events/events-team/content/meet-and-greet.md

```diff
@@ -0,0 +1,94 @@
+
+Some teams that are not SIGs or WGs may also participate, e.g. ClusterAPI or the CoCC.  There is no specific policy on this, deal with it on a case-by-case basis. Generally speaking, we want to say yes to teams that actually plan to show up as a group. In the rest of this document, "SIGs" should be considered to refer to "SIGs, WGs, and Teams".
+
+## Time and Location
+
+The M&G is generally held during lunch period on one of the main Kubecon days.  CNCF staff decides the exact schedule in order to not conflict with other lunchtime events.
+
+## Preparation & Timeline
+
+### 2-3 Months Before: Room request and SIG gear
+
+Request a large room for the M&G from the CNCF Contributor Summit Liaison.  This will be a large room, set with at least 10 round tables.
+
+The gear for the M&G consists of table sign stands for each SIG, and a bag of buttons for each.  Some of this gets used/goes missing each Kubecon, so ask CNCF staff to check that:
+
+- All currently active SIGs have at least a dozen buttons
+- All currently active SIGs have table signs, and non-broken table stands
+
+The CNCF staff must do this because they have possession of the case of SIG gear.  However, the MGC should help by giving them a shortlist of the SIGs most likely to staff the event, as well as checking with them to remove any obsolete SIGs or WGs.
+
+If Kubernetes has a new SIG this year, or a team that really wants to recruit, notify the staff as soon as possible so that they can order new gear for that group.
+
+### 1 Month Before: Recruit SIG staffers
+
+Prepare a sign-up sheet for SIG volunteers, usually a spreadsheet so that folks from SIGs can see who else has signed up for their SIG.  Then work with Contributor Comms in order to message all of the SIG leads.  Notify folks on the following channels:
+
+- Slack, #chairs-and-techleads
+- Slack, #contributor-summit
+- kubernetes-dev mailing list
+- LWKD
+- Ideally, a message to each of the primary #sig-* channels (this is a good task for a shadow, and should probably be bundled with outer communications)
+
+This solicitation might be bundled with other contributor-summit messaging that needs to go to all SIG leads, such as a call for booking SIG meeting rooms.
+
+Add a page in the Contributor Summit website for the Meet & Greet.
+
+# Follow-ups
+
+Check weekly to see who's signed up.  While participation by all SIGs is valuable, the following SIGs are particularly popular with 1st-time contributors, and if nobody signs up you should message them:  SIG-Docs, SIG-Release, SIG-CLI, SIG-Contributor Experience, SIG-K8s-Infra.
+
+### 1 Week Before: Solicit Attendees and Send Reminders
+
+By this time, CNCF events should have given you a room number and day. Send out a reminder to all of the chairs and techleads, and give them the updated information including the location and time.  Update the Summit website with the full information.
```

Comment:
The past few times, we've had last-minute cancellations due to company travel policies. Perhaps part of the week before should include ensuring everyone who signed up hasn't had any last-minute cancellations?

Reply:
Oh, yeah, after we merge this you can add something for "during kubecon".